### PR TITLE
Align contributors in rows

### PR DESCRIPTION
**Pull request by feimosi**
That will align them more evenly per row.

### CODE REVIEW

**Comment by kentcdodds** [comment]
Cool! Could you add screenshots to this pull request? Thanks!

**Comment by feimosi** [comment]
Sure, it makes each contributor get the same width in a row. Before, long names could take too much space. More or less it looks like this:

**Before:**


---

**After:**

File: src/pages/contributors/index.js [hunk 1]
```diff
@@ -99,5 +99,8 @@ Contributors.styles = StyleSheet.create({
     flexFlow: 'row wrap',
     textAlign: 'center',
   },
-  contributor: {margin: '0 10px 18px 10px'},
+  contributor: {
+    margin: '0 10px 18px 10px';
```
Comment (kentcdodds) [inline src/pages/contributors/index.js]: This is a syntax error. Remember, this is a JavaScript object, not a CSS file :)

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Love the screenshot! That'll look much better. Just fix the syntax error and we'll be good :)

File: src/pages/contributors/index.js (hunk 1 above)
Comment (feimosi) [reply src/pages/contributors/index.js]: My bad, sorry! I totally forgot myself :)

**Comment by kentcdodds** [comment]
Excellent! Thank you!

**Pull request merged**